pkg/client/connection.go

```diff
@@ -97,5 +103,25 @@ func transportCredentials(config *ApiConnectionDetails) grpc.DialOption {
 	if !config.ForceNoTls && !strings.Contains(config.ArmadaUrl, "localhost") {
 		return grpc.WithTransportCredentials(credentials.NewClientTLSFromCert(nil, ""))
 	}
-	return grpc.WithInsecure()
+	return grpc.WithTransportCredentials(insecure.NewCredentials())
+}
+
+func (a *ApiConnectionDetails) ArmadaRestServerHealthcheck() error {
+	url := a.ArmadaRestUrl
+	if url == "" {
+		return errors.New("armada server rest api url not provided")
```

Comment:
I prefer using capitalised Armada everywhere. Since it's a name I think that's more correct.